website/frontend/management/commands/scraper.py

```diff
@@ -263,9 +269,11 @@ def get_hash(version):
 
             number_equal = sum(1 for h in hashes if h == my_hash)
 
-            logger.debug('Got %s', number_equal)
+            logger.debug('Got %s previous version files have an identical hash', number_equal)
```

Comment:
s/have/with/, or maybe "Found %s previous commits of an identical file".  But even that's ugly, since often number_equal is 1 so commits shouldn't be plural...

Reply:
I think that should be okay for a logging message